src/components/WorkOrders/SummaryCloseWorkOrder.js

```diff
@@ -149,21 +149,32 @@ const SummaryCloseWorkOrder = ({
                         )}
                       </>
                     }
-                    handleClick={changeStep}
                   />
                 )}
 
                 <TableRow
                   label="Additional notes"
                   value={followOnData.additionalNotes}
-                  handleClick={changeStep}
                 />
               </TBody>
             </Table>
           </>
         )}
 
-        <PrimarySubmitButton label="Confirm and close" />
+        <div style={{ margin: '30px 0 0 0', display: 'flex' }}>
+          {' '}
+          <PrimarySubmitButton
```

Comment:
Not really a primary submit button if it's a secondary "go back" button. Could use another component if available or rename if it's not too disruptive?